core/storage/base_model/gae_models.py

```diff
@@ -304,6 +304,19 @@ def update_timestamps_multi(cls, entities, update_last_updated_time=True):
         datastore_services.update_timestamps_multi(
             entities, update_last_updated_time=update_last_updated_time)
 
+    @classmethod
+    @transaction_services.run_in_transaction_wrapper
+    def put_multi_transactional(cls, entities):
+        """Stores the given datastore_services.Model instances and runs it
+        through a transaction. Either all models are deleted, or none of them
```

Comment:
Should this say that either all models are put, or none of them are? This seems to be a put operation, not a deletion.

Reply:
Yes, totally missed this one, I will fix it asap.

Reply:
I went with ```Either all models are stored``` instead of ```put``` because that seems to be the case for the non-transactional ```put_multi``` documentation as well.

Reply:
This seems clear now--thanks!